components/landing/topCollections/utils/useCollectionsSales.ts

```diff
@@ -0,0 +1,29 @@
+import { CollectionSales } from './types'
+
+type CollectionsSalesResult = {
+  collectionsSales: CollectionSales[]
+}
+
+export const useCollectionsSales = (ids: string[]) => {
+  const CollectionSalesList = useState<CollectionSales[] | null>(
+    'CollectionSalesList'
+  )
+
+  const { data, error, loading } = useGraphql({
+    queryPrefix: 'rmrk',
```

Comment:
could we move this query to `subsquid` since it's used on other chains?

Reply:
sure thing